x-pack/packages/ml/trained_models_utils/src/constants/trained_models.ts

```diff
@@ -39,4 +39,34 @@ export const BUILT_IN_MODEL_TYPE = i18n.translate(
   { defaultMessage: 'built-in' }
 );
 
+export const CURATED_MODEL_TYPE = i18n.translate(
+  'xpack.ml.trainedModels.modelsList.curatedModelLabel',
+  { defaultMessage: 'curated' }
+);
+
 export const BUILT_IN_MODEL_TAG = 'prepackaged';
+
+export const CURATED_MODEL_TAG = 'curated';
+
+export const CURATED_MODEL_DEFINITIONS = {
+  '.elser_model_1_SNAPSHOT': {
```

Comment:
The `_SNAPSHOT` will need to be removed in good time for release. Please set up a calendar reminder to do this. I will try to remember too, but the more people who are thinking about the need to update these places the better.